gi_wrapper_cpp/wrapper_logger.cpp

```diff
@@ -30,7 +30,7 @@ const string FILENAME = "log.txt";
 string createWrapperTimestamp()
 {
 	time_t rawtime;
-	struct tm * timeinfo;
+	struct tm* timeinfo;
```

Comment:
@kdbogusz please change old linux-style timestamp taking to newer more portable way of doing that - see https://en.cppreference.com/w/cpp/chrono